### DISCUSSION TITLE
[委托] 树上长猫了啊啊啊啊

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#1393] [其他]
![1723362970124](https://github.com/user-attachments/assets/96249f95-46cc-4e36-b855-100d1a6a3fa6)
好可怕好可怕好可怕…六分街的树上怎么会有猫咪啊啊啊啊。毛茸茸的生物真的好可怕，但这只猫下不来的话可能会造成潜在危险…

怎么办怎么办，最好快来个不怕猫的人帮帮我。

地点：六分街，治安局会颁发“友好市民”奖项 （真的不是钓鱼执法啊啊啊）
![1723330814554](https://github.com/user-attachments/assets/f26e8861-c0cd-4e98-b3aa-b02bd646d084)

**Comment by lllliufeng** [comment]
我滴，来辣

**Comment by 127kzzz** [comment]
这个委托我接了

**Comment by EhuanyingK** [comment]
别抢，这个委托是我的

**Comment by gaojiaobei** [comment]
我有相关委托的经验，交给我吧

**Comment by time-p** [comment]
这个委托让我来！

**Comment by RMWS1226** [comment]
让青衣帮你罢

**Comment by wushanef** [comment]
委托是你们的，朱鸢长官是我nina的了

**Reply by ZhuYuan-148** [reply]
你说，我在听～

**Reply by wushanef** [reply]
别抓我，我才16，未成年~凸(^▽^)凸

**Comment by ZhuYuan-148** [comment]
谢谢各位的帮助，猫已经顺利下来了（话说它是怎么上去的呢…）
